hack/jenkins/build_iso.sh

```diff
@@ -65,16 +65,20 @@ else
 	export ISO_BUCKET
 fi
 
-make release-iso | tee iso-logs.txt
-# Abort with error message if above command failed
-ec=$?
-if [ $ec -gt 0 ]; then
-	if [ "$release" = false ]; then
-		gh pr comment ${ghprbPullId} --body "Hi ${ghprbPullAuthorLoginMention}, building a new ISO failed.
-		See the logs at: https://storage.cloud.google.com/minikube-builds/logs/${ghprbPullId}/${ghprbActualCommit::7}/iso_build.txt
-		"
-	fi
-	exit $ec
+if ! make release-iso 2>&1 | tee iso-logs.txt; then
+  # Exit of `make` (PIPESTATUS[0]); fallback to 1 if unavailable
+  ec=${PIPESTATUS[0]:-1}
+
+  # Only comment on non-release; default release=false if unset
+  if [[ ${release:-false} != "true" ]]; then
```

Comment:
Preview code was simpler but could be better with quoting.

`#{release:-false}` is not needed since we define the release variable in line 56. If it is not defined in all cases we should fix the issue by always defining it and avoiding expansion like this. This makes the code less readable and promote bad code using undefined variables.